CIPs/cip-0051.md

```diff
@@ -53,7 +53,7 @@ In the future, we plan to allow for issuers to create attestations for the fact
 
 ### Issuers
 
-Issuers should create a separate signer key that is associated with their account for the express purpose of signing verifiable credentials. The `AttestationSigner` role should be assigned to this signer key. It is possible to authorize multiple signer keys for this role.
+Issuers should create a separate signer key that is associated with their account for the express purpose of signing attestations. The `AttestationSigner` role should be assigned to this signer key. It is possible to authorize multiple signer keys for this role.
```

Comment:
@alecps: Made this tiny change to fix what looks like a typo. 
Let me know if you disagree 🙏

Reply:
I agree 👍 That's why I replaced "verifiable credentials" with "attestations".

Reply:
oh woops yes true thank you haha